src/orchestrator-kubernetes/src/lib.rs

```diff
@@ -790,6 +792,15 @@ impl NamespacedOrchestrator for NamespacedKubernetesOrchestrator {
             None
         };
 
+        let security_context = if let Some(fs_group) = self.config.service_fs_group {
+            Some(PodSecurityContext {
+                fs_group: Some(fs_group),
+                ..Default::default()
+            })
```

Comment:
This code is not covered by any test case. Any changes of covering it?

Reply:
There is really no way to test this, but I am going to test it in staging!